include/nbl/builtin/hlsl/subgroup/arithmetic_portability.hlsl

```diff
@@ -0,0 +1,95 @@
+#ifndef _NBL_BUILTIN_HLSL_SUBGROUP_ARITHMETIC_PORTABILITY_INCLUDED_
+#define _NBL_BUILTIN_HLSL_SUBGROUP_ARITHMETIC_PORTABILITY_INCLUDED_
+
+#ifndef NBL_GL_KHR_shader_subgroup_arithmetic
+#include <nbl/builtin/hlsl/subgroup/basic_portability.hlsl>
+#endif
+
+static uint4 WHOLE_WAVE = ~0;
```

Comment:
`const` also why do you define it here?

Reply:
Hmm I'm not really familiar with the HLSL compiler tricks yet so I'm not sure if something like `~0` should be defined once as static or be inlined was kinda waiting for a confirmation. Additionally my understanding is that with HLSL we want to avoid #defines whenever we can and replace them with static values.

Reply:
#define and #undef at the end after you don't need it